Polygon.js

```diff
@@ -0,0 +1,167 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?:\\/\\/(www.)?polygon.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-12-06 15:31:27"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2015 czar
+    http://en.wikipedia.org/wiki/User_talk:Czar
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function pageType (doc, url) { 
+	return ZU.xpathText(doc, '//script[contains(text(), "page_type :")]').split('page_type : \"')[1].split(/\"/)[0];
```

Comment:
kind of the same issue as below. This throws an error when this evaluates to null. I actually did find an example for this:
http://www.polygon.com/rss/index.xml
While this isn't fatal (detectWeb also just returns false when it throws an error) we still want to avoid this, so have the `pageType` function return null when `ZU.xpathText` returns null.